patches/node/chore_remove_allcan_read_write_from_setaccessor.patch

```diff
@@ -0,0 +1,60 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Shelley Vohr <shelley.vohr@gmail.com>
+Date: Wed, 15 Nov 2023 12:09:38 +0100
+Subject: chore: remove AllCan Read/Write from SetAccessor
+
+Refs https://chromium-review.googlesource.com/c/v8/v8/+/5006387
+
+This patch can be removed when Node.js upgrades to a version of V8
```

Comment:
You can backport this change to reduce future conflicts: https://github.com/nodejs/node/pull/50709